### PR TITLE
delete design

### PR DESCRIPTION
**Pull request by RaviKhandavilli**
Added code to delete design.
Renamed files to maintain consistent naming conventions.

### CODE REVIEW

**Review by Divlik — DISMISSED** [review]

File: cmd/flamectl/cmd/delete_design.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/cisco-open/flame/cmd/flamectl/resources/design"
+	"github.com/spf13/cobra"
```
Comment (myungjin) [inline cmd/flamectl/cmd/delete_design.go:20]: Please follow https://github.com/cisco-open/flame/blob/main/CONTRIBUTING.md#use-a-consistent-coding-style

File: cmd/flamectl/cmd/delete_design.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Cisco Systems, Inc. and its affiliates
```
Comment (myungjin) [inline cmd/flamectl/cmd/delete_design.go:1]: please rename file name as "remove_design.go". This is for consistency and grouping of file names based on their functionality.

File: cmd/flamectl/cmd/delete_design.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		checkInsecure(cmd)
+
+		designId := args[0]
+
+		params := design.Params{}
+		params.Endpoint = config.ApiServer.Endpoint
+		params.User = config.User
+		params.DesignId = designId
+
+		return design.Delete(params)
```
Comment (myungjin) [inline cmd/flamectl/cmd/delete_design.go:38]: rename it as "Remove"

File: cmd/flamectl/resources/design/design.go [hunk 4]
```diff
@@ -61,6 +61,39 @@ func Create(params Params) error {
 	return nil
 }
 
+func Delete(params Params) error {
```
Comment (myungjin) [inline cmd/flamectl/resources/design/design.go:64]: Please rename it as "Remove"

File: pkg/openapi/controller/api_designs_service.go [hunk 5]
```diff
@@ -27,12 +27,12 @@ package controller
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"net/http"
 
 	"github.com/cisco-open/flame/cmd/controller/app/database"
 	"github.com/cisco-open/flame/pkg/openapi"
+	"go.uber.org/zap"
```
Comment (myungjin) [inline pkg/openapi/controller/api_designs_service.go:35]: Please follow https://github.com/cisco-open/flame/blob/main/CONTRIBUTING.md#use-a-consistent-coding-style

**Review by myungjin — COMMENTED** [review]
left a few comments.

File: cmd/flamectl/cmd/delete_design.go (hunk 1 above)
Comment (RaviKhandavilli) [reply cmd/flamectl/cmd/delete_design.go]: done

File: cmd/flamectl/cmd/delete_design.go (hunk 3 above)
Comment (RaviKhandavilli) [reply cmd/flamectl/cmd/delete_design.go]: done

File: pkg/openapi/controller/api_designs_service.go (hunk 5 above)
Comment (RaviKhandavilli) [reply pkg/openapi/controller/api_designs_service.go:36]: done

File: cmd/flamectl/resources/design/design.go (hunk 4 above)
Comment (RaviKhandavilli) [reply cmd/flamectl/resources/design/design.go]: done

File: cmd/flamectl/cmd/delete_design.go (hunk 2 above)
Comment (RaviKhandavilli) [reply cmd/flamectl/cmd/delete_design.go]: done

File: cmd/flamectl/cmd/remove_design.go [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/cisco-open/flame/cmd/flamectl/resources/design"
+
+	"github.com/spf13/cobra"
```
Comment (myungjin) [inline cmd/flamectl/cmd/remove_design.go:21]: Please look at https://github.com/cisco-open/flame/blob/main/CONTRIBUTING.md#use-a-consistent-coding-style

File: pkg/openapi/controller/api_designs_service.go [hunk 7]
```diff
@@ -27,12 +27,13 @@ package controller
...
 import (
 	"context"
-	"errors"
 	"fmt"
 	"net/http"
 
 	"github.com/cisco-open/flame/cmd/controller/app/database"
 	"github.com/cisco-open/flame/pkg/openapi"
+
+	"go.uber.org/zap"
```
Comment (myungjin) [inline pkg/openapi/controller/api_designs_service.go:36]: Please look at it carefully: https://github.com/cisco-open/flame/blob/main/CONTRIBUTING.md#use-a-consistent-coding-style

**Review by myungjin — COMMENTED** [review]
comments left.

File: cmd/flamectl/cmd/remove_design.go (hunk 6 above)
Comment (RaviKhandavilli) [reply cmd/flamectl/cmd/remove_design.go:19]: done

File: pkg/openapi/controller/api_designs_service.go (hunk 7 above)
Comment (RaviKhandavilli) [reply pkg/openapi/controller/api_designs_service.go]: done

**Review by myungjin — APPROVED** [review]
lgtm

**Pull request merged**